Modules/IndividualAssessment/classes/class.ilObjIndividualAssessment.php

```diff
@@ -7,36 +7,72 @@
  * @author Denis Klöpfer <denis.kloepfer@concepts-and-training.de>
  */
 
+declare(strict_types=1);
 
 require_once 'Services/Object/classes/class.ilObject.php';
 require_once 'Modules/IndividualAssessment/classes/Settings/class.ilIndividualAssessmentSettings.php';
 require_once 'Modules/IndividualAssessment/classes/Settings/class.ilIndividualAssessmentSettingsStorageDB.php';
 require_once 'Modules/IndividualAssessment/classes/Members/class.ilIndividualAssessmentMembersStorageDB.php';
 require_once 'Modules/IndividualAssessment/classes/AccessControl/class.ilIndividualAssessmentAccessHandler.php';
 require_once 'Modules/IndividualAssessment/classes/FileStorage/class.ilIndividualAssessmentFileStorage.php';
-class ilObjIndividualAssessment extends ilObject {
 
+class ilObjIndividualAssessment extends ilObject
+{
 	protected $lp_active = null;
 
-	public function __construct($a_id = 0, $a_call_by_reference = true) {
+	/**
+	 * @var string
+	 */
+	protected $type;
```

Comment:
Was this automatically added by your IDE? Doesn't `ilObject` already have a property like this?